app/src/main/java/com/phivle/fitnesstasks/MainActivity.java

```diff
@@ -0,0 +1,28 @@
+package com.phivle.fitnesstasks;
+
+import android.os.Bundle;
+import android.support.v4.view.ViewPager;
+import android.support.design.widget.TabLayout;
+import android.support.v7.app.AppCompatActivity;
+
+public class MainActivity extends AppCompatActivity {
+
+    private SectionsPagerAdapter mSectionsPagerAdapter;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_calendar);
+
+        mSectionsPagerAdapter = new SectionsPagerAdapter(getSupportFragmentManager(), this);
+        //set the view pager
```

Comment:
i'm in favor of not committing these psuedocode comments to the repo. We should keep comments for lines of code that are not explicit in what they are doing (such as necessary empty constructors)